library/common/extensions/key_value/platform/config.h

```diff
@@ -32,15 +32,15 @@ class PlatformKeyValueStore : public KeyValueStoreBase {
   void flush() override;
 
 private:
+  // TODO(alyssawilk, goaway) the default PlatformInterface should do up calls through Java and this
+  // can be moved to a non-optional reference.
```

Comment:
this is a non-optional reference :-)
Maybe remove this TODO and add one by Api::External::retrieveApi to gracefully handle the API not being registered?

Reply:
Oops yes, done!